akka-persistence-query/src/main/scala/akka/persistence/query/Offset.scala

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2015-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.persistence.query
+
+import java.util.UUID
+
+trait Offset {
+  def hashCode: Int
+}
+
+final case class Sequence(val value: Long) extends Offset with Ordered[Sequence] {
+
+  override def hashCode: Int = value.hashCode
+
+  override def toString: String = value.toString
```

Comment:
Yes, that makes more sense. I didn't realize hasCode was provided by case class, and it looks case class's toString is better than what I did - removed them in the new commit.